### ISSUE TITLE
Pretrained  Action Detection on AVA-Kinetics model weights

### ISSUE THREAD

**Issue by girmaji08** [OpenGVLab/VideoMAEv2#29]
Hello,
I'm Rohit. I'm a researcher from IIIT-Hyderabad,India. I am working on a project related to Action Detection. Can you please provide your pretrained model weights on AVA-Kinetics/AVAv2.2. It would really help me in building a model that uses these pretrained models weights and also in ablation studies.

I filled your form for model weights. But I am getting model weights for video action classification which is not what I want. I want pretrained model weights for action detection

Thanks,
Rohit

**Comment by girmaji08** [comment]
Hello,
Can I have pretrained model weights for AVA/AVA-KINETICS.
Thanks,
Rohit Girmaji

**Comment by congee524** [comment]
Hello, I encountered severe overfitting when training AVA and AVA-Kinetics and the model has reached optimal performance in the early stages.

But I am losing the validation results of AVA. I'll provide you with the AVA-Kinetics weights first (which will show higher results than the paper reporting points, since I missed the camera-ready ddl).

I will retest the results on AVA later when I have time.

Alternatively, you can consider training the model yourself according to the setup in the appendix of the paper, and that hyperparameter should train the model better than I did.

**Comment by shivanimall** [comment]
Hello @congee524 I am also hoping to obtain access to pretrained model weights for AVA/AVA-KINETICS, wondering if you can share it with me as well? I submitted a request via your google form today.

**Comment by congee524** [comment]
> Hello @congee524 I am also hoping to obtain access to pretrained model weights for AVA/AVA-KINETICS, wondering if you can share it with me as well? I submitted a request via your google form today.

My google drive does not have enough space to store this model. Is there any way I can share it with you?

**Comment by congee524** [comment]
> Thanks a lot @congee524 I have unlimited storage space in one of my google drives, if I added you to that space, would you be able to upload it there? Let me know what is the best email to use.

congee524@gmail.com

I may need to look for the model weights and probably upload it tonight or tomorrow :p

**Comment by shivanimall** [comment]
added you, and thanks bunch!!

**Comment by congee524** [comment]
I uploaded AVA-Kinetics model weigh. I can't find the best  ava mode (Cuz I lost the logs, need to re-test the accuracy of all the models and don't have the resources to do that at the moment). Hope it helps :)

**Comment by shivanimall** [comment]
thanks a lot, I see it :)

**Comment by ercanburak** [comment]
Hello @shivanimall and @congee524, can you share the AVA-Kinetics model weights with me (burakercan@hacettepe.edu.tr) as well? (or to the public maybe?)

**Comment by Jun-hulahula** [comment]
Hello，@congee524, can you share the AVA/AVA-Kinetics model weights with me as well? 
It would really help me in building a model that uses these pretrained models weights and also in ablation studies.
Thanks,
Email id: wenjj26@mail2.sysu.edu.cn

**Comment by congee524** [comment]
Sure, I will share it tonight. If I forget, please ping me tomorrow.

**Comment by ercanburak** [comment]
Hello @congee524, can you share the AVA-Kinetics model weights with me ([burakercan@hacettepe.edu.tr](mailto:burakercan@hacettepe.edu.tr)) as well?

**Comment by Jun-hulahula** [comment]
> Sure, I will share it tonight. If I forget, please ping me tomorrow.

Hello，I will appreciate it if you can share me today!
Have a good day!

**Comment by congee524** [comment]
[vit_g_ps14_ak_ft_ckpt_7_clean.pth](https://drive.google.com/file/d/1kI7DQRj12SZ8xIsE9q-mACDSaR1sIAMK/view?usp=sharing)

Hope it helps :p

**Comment by Vincia-Jun** [comment]
> [vit_g_ps14_ak_ft_ckpt_7_clean.pth](https://drive.google.com/file/d/1kI7DQRj12SZ8xIsE9q-mACDSaR1sIAMK/view?usp=sharing)
> 
> Hope it helps :p

Thinks！
I will appreciate it if you can explain the name of the document ?
g means "giant"?
what's "ak" and "ft", as well as "7" ?

**Comment by Vincia-Jun** [comment]
> [vit_g_ps14_ak_ft_ckpt_7_clean.pth](https://drive.google.com/file/d/1kI7DQRj12SZ8xIsE9q-mACDSaR1sIAMK/view?usp=sharing)
> 
> Hope it helps :p

Thanks！
I will appreciate it if you can explain the name of the document ?
g means "giant"?
what's "ak" and "ft", as well as "7"?

**Comment by congee524** [comment]
> > [vit_g_ps14_ak_ft_ckpt_7_clean.pth](https://drive.google.com/file/d/1kI7DQRj12SZ8xIsE9q-mACDSaR1sIAMK/view?usp=sharing)
> > Hope it helps :p
> 
> Thanks！ I will appreciate it if you can explain the name of the document ? g means "giant"? what's "ak" and "ft", as well as "7"?

vit_g -> vit-giant
ak -> ava-kinetics
ft -> finetune
ckpt_7 -> the checkpoint at epoch 7

**Comment by payne4handsome** [comment]
@congee524 
Hello! Thank you for the great work!

Can you provide code and scripts to finetune on AVA-Kinetis or AVAV2.2. I want finetune VideoMAEV2 on my own datasets. I can find related scripts in [VideoMAE-Action-Detection](https://github.com/MCG-NJU/VideoMAE-Action-Detection), but it based VideoMAE V1. 

thanks a lot.
